src/main/java/commands/DeleteAuthorCommand.java

```diff
@@ -4,13 +4,15 @@
 import author.AuthorList;
 import exceptions.TantouException;
 import ui.Ui;
-import java.util.logging.*;
+
+import java.util.logging.Level;
+import java.util.logging.Logger;
```

Comment:
Good job on catching this guideline violation! It's better to explicitly specify which methods we are importing from a package.